src/_includes/get-sdk.md

```diff
@@ -1,5 +1,11 @@
 Once you’re ready to move beyond DartPad and develop real apps,
-you need the Dart SDK.
+you need an SDK.
+You can either download the Dart SDK directly
+(as described below)
+or [download the Flutter SDK,][]
+which (as of Flutter 1.20) includes the Dart SDK.
```

Comment:
This surprises me. I thought it was as of 1.17.

Reply:
@kevmoo can you confirm the first version of Flutter that includes the full Dart SDK?

Reply:
This is new to me, too! @timsneath ?

Reply:
Technically, the full Dart SDK isn't in the Flutter SDK even in 1.20, since `dart2native` (e.g.) isn't in the Flutter SDK. Its functionality will be available when the expanded `dart` command is available, which looks like it'll be 1.21. I'm going to create a PR to undo the work of this PR.

Reply:
For now, I'm just changing 1.20 to 1.21. I hope that isn't too confusing. #2562